ssz_serialization.nimble

```diff
@@ -20,8 +25,8 @@ proc test(env, path: string) =
   if not dirExists "build":
     mkDir "build"
   exec "nim " & lang & " " & env &
-    " -r --hints:off --warnings:off " & path
+    " -r --hints:off --warnings:on " & path
 
 task test, "Run all tests":
-  test "--threads:off", "tests/test_all"
-  test "--threads:on", "tests/test_all"
+  test "--threads:off -d:PREFER_BLST_SHA256=false", "tests/test_all"
```

Comment:
Building it currently only using the sha256 implementation of nimcrypto as I'm having issues with BLST outside of the nimbus-eth2 env.